resources/log.go

```diff
@@ -13,15 +13,5 @@ func Log() *schema.Table {
 		Resolver:  fetchLog,
 		Transform: transformers.TransformWithStruct(rest.Log{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```

Comment:
Removing as https://docs.swetrix.com/statistics-api#get-v1log doesn't return a `project_id` or any unique key in the response